main.tf

```diff
@@ -43,7 +43,7 @@ resource "google_project_service" "cloudsql_api" {
 
 module "google_mysql_db" {
   source                          = "GoogleCloudPlatform/sql-db/google//modules/mysql"
-  version                         = "9.0.0"
+  version                         = "13.0.0"
```

Comment:
Has this been tested in dev/stg if it will end up breaking anything else?

I  believe `insights_config` is already available in v9.0.0. Need to test this out before we can merge this.

Reply:
Hi Rakib, this has been rested in development and it works as expected